service/src/main/java/com/commercetools/pspadapter/payone/domain/payone/PayonePostServiceImpl.java

```diff
@@ -57,15 +57,9 @@ public static PayonePostServiceImpl of(final String payoneServerApiUrl) throws I
                     getNameValuePairsWithExpandedLists(baseRequest.toStringMap(false));
 
             String serverResponse = executePostRequestToString(this.serverAPIURL, mappedListParameters);
-
-            if (serverResponse.contains("status=ERROR")) {
-                LOG.error("-> Payone POST request parameters: {}",
-                        getNameValuePairsWithExpandedLists(baseRequest.toStringMap(true)).toString());
-                LOG.error("Payone POST response: {}", serverResponse);
-            }
```

Comment:
I think POST response might be useful or the idea is that one can look it up on CTP payment object?